target/linux/generic/pending-5.4/681-NET-add-of_get_mac_address_mtd.patch

```diff
@@ -79,7 +79,7 @@ Signed-off-by: Felix Fietkau <nbd@nbd.name>
 +	if (inc_idx > 5)
 +		return NULL;
 +
-+	if (!of_property_read_u32(np, "mtd-mac-address-increment", &mac_inc))
++	if (!of_property_read_s32(np, "mtd-mac-address-increment", &mac_inc))
```

Comment:
Why has this been working already on ath79?

Reply:
Don't really know... I just notice that in some ath79 dts we have negative value but looking at the code isn't this wrong?  
Since mac_inc is u32 and proprety_read treat the value as unsigned i can't really find where the negative value is converted to unsigned... Could be that nobody notice that?

Reply:
Since I put some effort into having MAC addresses correct, I have discussed so many cases in PRs that I can't imagine negative increments not working on ath79.

Reply:
Actually.... I can make a quick test with my router and check if the value is decremented correctly